README.md

```diff
@@ -479,6 +479,39 @@ Returns the [`PlacesService`](google.maps.places.PlacesService) class to use dir
 google.maps.places.PlacesService
 ```
 
+### useElevationService
+
+React hook to use the [Elevation Service](https://developers.google.com/maps/documentation/javascript/elevation) in any component.
+
+#### Usage
+
+```jsx
+import React from 'react';
+import {useElevationService} from '@ubilabs/google-maps-react-hooks';
+
+const MyComponent = () => {
+  const elevator = useElevationService();
+
+  elevator?.getElevationForLocations(
+    {locations: [location]},
+    (results: google.maps.ElevationResult[]) => {
+      // Do something with results
+    }
+  );
```

Comment:
Maybe this makes it clearer, that a `location` is passed and what it is.
```suggestion
```tsx
import React, {useEffect} from 'react';
import {useElevationService} from '@ubilabs/google-maps-react-hooks';

const MyComponent = () => {
  const elevator = useElevationService();
  const location = /** google.maps.LatLng */;

  useEffect(() => {
    elevator?.getElevationForLocations(
      {locations: [location]},
      (results: google.maps.ElevationResult[]) => {
        // Do something with results
      }
    );
  }, [location]);
```

Reply:
Yes, very good idea